tests/output/dbus/DbusDocumentTest.py

```diff
@@ -83,5 +83,5 @@ def test_analyze(self):
         uut.SetConfigFile(self.config_path + "2")
         output = uut.Analyze()
         self.assertEqual(output[0], 255)
-        self.assertEqual(output[1][1]["log_level"], "ERROR")
-        self.assertEqual(output[1][1]["message"], Constants.CRASH_MESSAGE)
+        self.assertEqual(output[1][0]["log_level"], "ERROR")
+        self.assertEqual(output[1][0]["message"], Constants.CRASH_MESSAGE)
```

Comment:
That's interesting, I don't see any reason for this to have changed.
Changing it from 1 -> 0 means that the second log message is now the first log message. Was this changed in recent code?
It may be a good idea to do

import pprint from pprint
pprint (output)

And check which error message is not there now